examples/integration-tutorials/routing-requests-based-on-message-content/guide/content_based_routing.bal

```diff
@@ -0,0 +1,55 @@
+import ballerina/http;
+import ballerina/log;
+
+@http:ServiceConfig {
+    basePath: "/healthcare"
+}
+service contentBasedRouting on new http:Listener(9080) {
```

Comment:
We need to add the variable and method comments

Reply:
Added